src/main/kotlin/no/nav/syfo/oppfolgingstilfelle/bit/cronjob/OppfolgingstilfelleCronjob.kt

```diff
@@ -34,7 +34,9 @@ class OppfolgingstilfelleCronjob(
                         personIdentNumber = oppfolgingstilfelleBit.personIdentNumber,
                     ).toOppfolgingstilfelleBitList().toMutableList()
 
-                    oppfolgingstilfelleBitForPersonList.add(0, oppfolgingstilfelleBit)
+                    if (!connection.isTilfelleBitAvbrutt(oppfolgingstilfelleBit.uuid)) {
+                        oppfolgingstilfelleBitForPersonList.add(0, oppfolgingstilfelleBit)
```

Comment:
Kanskje denne `add`-funksjonen kunne brukt named params? Da hadde det vært litt lettere å forstå at man sender index for hvor man skal legge inn biten i lista først.